crest/Assets/Crest/Crest/Scripts/LodData/Settings/SimSettingsAnimatedWaves.cs

```diff
@@ -50,6 +51,9 @@ public enum CollisionSources
         [Tooltip("The render texture format to use for the wave simulation. It should only be changed if you need more precision. See the documentation for information.")]
         public GraphicsFormat _renderTextureGraphicsFormat = GraphicsFormat.R16G16B16A16_SFloat;
 
+        [Predicated("_collisionSource", true, 3), DecoratedField]
+        public FFTBakedData _bakedFFTData;
+
         public override void AddToSettingsHash(ref int settingsHash)
         {
             base.AddToSettingsHash(ref settingsHash);
```

Comment:
Might as well add the collision source to the hash now that it is relevant in the editor again with this PR.

Reply:
good shout, done